resources/build/build-utils.sh

```diff
@@ -141,12 +141,28 @@ function printVersionUtilsDebug() {
     echo "VERSION_WITH_TAG: $VERSION_WITH_TAG"
 }
 
+function findShouldSentryRelease() {
+    # Default, for 'test' or 'local' environment, or in case $VERSION_ENVIRONMENT is improperly specified.
+    # (May be overridden by -upload-sentry in supporting build scripts.)
+    UPLOAD_SENTRY=false
+
+    # Default: override to `true` for release builds.
+    case $VERSION_ENVIRONMENT in
+    # Actual release tiers
+    alpha | beta | stable)
+        UPLOAD_SENTRY=true
+        ;;
+    esac
+}
+
 findRepositoryRoot
 findTier
 findVersion
 # printVersionUtilsDebug
 printBuildNumberForTeamCity
 
+findShouldSentryRelease
```

Comment:
Please add `$UPLOAD_SENTRY` to list of variables set in the comments at top of this script.